google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -114,6 +135,23 @@ private Builder() {
               .build());
 
       // Per-method settings using baseSettings for defaults.
+      readRowsSettings = ServerStreamingCallSettings.newBuilder();
+      /* TODO: copy timeouts, retryCodes & retrySettings from baseSettings.readRows once it exists in GAPIC */
```

Comment:
Does this exist now?

Reply:
Not yet, this will need the toolkit to get updated to generate the correct retry settings for streaming methods